src/widgets/tdarr/HealthCheckStatus.tsx

```diff
@@ -0,0 +1,105 @@
+import {
+  ActionIcon,
+  Divider,
+  Group,
+  HoverCard,
+  Indicator,
+  MantineColor,
+  RingProgress,
+  Stack,
+  Text,
+} from '@mantine/core';
+import { IconHeartbeat } from '@tabler/icons-react';
+import { useTranslation } from 'next-i18next';
+import { useColorScheme } from '~/hooks/use-colorscheme';
+import { TdarrStatistics } from '~/server/api/routers/tdarr';
+
+interface StatisticsBadgeProps {
+  statistics?: TdarrStatistics;
+}
+
+export function HealthCheckStatus(props: StatisticsBadgeProps) {
+  const { statistics } = props;
+
+  const { colorScheme } = useColorScheme();
+  const { t } = useTranslation('modules/tdarr-queue');
+
+  if (!statistics) {
+    return (
+      <ActionIcon loading>
+        <IconHeartbeat size={20} />
+      </ActionIcon>
+    );
+  }
+
+  const indicatorColor = statistics.failedHealthCheckCount
+    ? 'red'
+    : statistics.stagedHealthCheckCount
+      ? 'yellow'
+      : 'green';
+
+  return (
+    <HoverCard position="bottom" width={250} shadow="sm" openDelay={0} closeDelay={0}>
```

Comment:
Since it's a modal, it needs to be of a slightly different background color.
Remove delays.